app/javascript/hyacinth_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldEdit.js

```diff
@@ -28,13 +28,16 @@ export default class EnabledDynamicFieldsEdit extends React.Component {
   render() {
     const { match: { params: { projectStringKey, digitalObjectType } } } = this.props;
 
+    // ALL CAPS enum value per graphql API
+    const digitalObjectTypeUcase = digitalObjectType.toUpperCase();
```

Comment:
Same thought here as earlier comment about expecting digitalObjectType to come back from the graphql api as upper case.